src/org/centos/pipeline/PipelineUtils.groovy

```diff
@@ -1,6 +1,17 @@
 #!/usr/bin/groovy
 package org.centos.pipeline
 
+env.cicoPipelineLibRepo = env.cicoPipelineLibRepo ?: 'CentOS/cico-pipeline-library'
```

Comment:
These should be the variable names so when they get populated by a PR they get overwritten.
env.ghprbGhRepository
env.ghprbActualCommit